warrior/Warrior

```diff
@@ -114,9 +115,7 @@ def main(parameter_list, a_defects, cse_execution, iron_claw, jiraproj, overwrit
                 print_info('Absolute path: {0}'.format(abs_filepath))
                 if Utils.file_Utils.fileExists(abs_filepath):
                     if len(overwrite.items()) > 0:
-                        default_repo = overwrite
```

Comment:
The reason why the original code won't work is because
default repo and overwrite both to the same dict which eventually become data_repo
when data_repo is updated during the first execution
its content isn't cleared when executing the 2nd test file from cmd line